cli/cmd/cluster.go

```diff
@@ -564,3 +627,13 @@ func parseAppStr(cmdCtx *cmdcontext.CmdCtx, appStr string) (string, string, stri
 
 	return configPath, appName, instName, nil
 }
+
+// checkRoleFlags checks that flags from 'cluster rs roles add' command
+// have correct values.
+func checkRoleFlags() error {
```

Comment:
The name does not match the description.
```suggestion
// checkRoleFlags checks that flags from 'cluster rs roles add' command
// have correct values.
func checkRolesAddFlags() error {
```

Reply:
Corrected.